### DISCUSSION TITLE
Autobrakes Level

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#5438] [Help] [answered]
Where can I find XMLVAR_Autobrakes_Level now, because this LVAR is gone?

**Comment by MStallen** [comment, marked as answer]
Found it in : A32NX_AUTOBRAKES_ARMED_MODE

**Comment by tracernz** [comment]
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md
